tests/ui/parser/ternary_operator.rs

```diff
@@ -0,0 +1,38 @@
+#![no_main]
+
+fn a() {
+    let x = 5 > 2 ? true : false; //~ ERROR Rust has no ternary operator
```

Comment:
nit: `//~^` is easier to read when chained all the way down

```suggestion
    let x = 5 > 2 ? true : false;
    //~^ ERROR Rust has no ternary operator
```